docs/dev/framework/caching.md

```diff
@@ -217,24 +217,104 @@ Moreover, you don't have to register to all the different callbacks such as `ons
 You can register to the [`oninvalidate_cache_tags` callback][5] and add your own tags.
 
 
-## Fragments and Edge Side Includes
+## Caching Fragments
 
 In Contao, content elements and front end modules can be implemented as so called
-_fragment controllers_. These fragments are then rendered with their defined renderer 
-and merged into the main content. Each fragment can also provide its own response
-and thus define whether it can be cached or not. If a fragment returns a response
-with a `Cache-Control: private` header for example, then the page on which the fragment
-is visible cannot be cached. On the other hand, if the fragment can be cached, it
-can provide its own cache tags, as mentioned previously.
-
-Contao brings its own `forward` fragment renderer, which provides the fragment with 
-a full clone of the request. This provides the fragment with the full `POST` data
-for example, contrary to Symfony's default `inline` renderer.
-
-The renderer can also be set to `esi`. In that case, if Symfony detects that it 
-is talking to  a gateway cache that supports ESI (like Symfony's built in reverse 
-proxy, that Contao uses), it generates an ESI include tag. See also [Symfony's documentation][esi] 
-on _Edge Side Includes_.
+_fragment controllers_. These fragments are then rendered with their defined renderer
+and merged into the main content. Each fragment returns a response and thus can
+tell whether it can be cached or not.
+
+
+There are two fundamentally different ways to render fragments:
+ 1. **Inline:** Inside the main page request and merged with the content before caching. In Symfony, this is handled
+    by the `inline` renderer, but Contao provides its own default `forward` renderer, which copies the original
+    request (whereas Symfony does not) to e.g. pass POST data to the subrequest.
+ 2. **Subrequest:** By generating an (internal) URL for the fragment, the fragment content is merged **after** caching
+    the page, which allows a fragment to have a different cache time than the main page. The `esi` renderer tells
+    the reverse proxy (cache) to fetch multiple URLs on a request and merge fragments before delivering to the client.
+    There's also an `hinclude` renderer, which merges fragments in the browser using JavaScript, but there's hardly
+    any use case for this anymore today.
+
+
+### Inline Fragments
+
+{{% notice note %}}
+Before **Contao 4.9**, inline fragments cannot affect the cache time of the page response.
+{{% /notice %}}
+
+By default, fragments like frontend modules and content elements are rendered directly inside the main request.
+This means the content of an inline fragment will be cached for as long as the page is cached, per definition
```

Comment:
`as configured in the page settings` maybe?

Reply:
Changed in 2ccd25f5de2e2ddd51ca8fff366caa726cc220e2